CONFIGURATION.md

```diff
@@ -221,6 +222,37 @@ tls_config:
 
 ```
 
+### `<unix_probe>`
+
+```yml
+
+# The query sent in the unix socket probe and the expected associated response.
```

Comment:
can you update the readme to mention unix_probe, and also add a line or two about permissions.

My understanding is that you need to be part of the same group (_for example: probing docker socket_), otherwise you will get permissions errors.

so imo, it's better to document the permissions, otherwise we will have users trying to use, and then fail.